TechTalk.JiraRestClient/FieldAttribute.cs

```diff
@@ -0,0 +1,14 @@
+namespace TechTalk.JiraRestClient
+{

```

Comment:
Reason is Stylecop. See here: http://stackoverflow.com/questions/125319/should-using-statements-be-inside-or-outside-the-namespace
But I don't care that much, so I can change that if this is a consistency need.